curvefs/conf/metaserver.conf

```diff
@@ -307,3 +307,9 @@ metaCacheOpt.metacacheGetLeaderRetry=3
 metaCacheOpt.metacacheRPCRetryIntervalUS=100000
 # RPC timeout of get leader
 metaCacheOpt.metacacheGetLeaderRPCTimeOutMS=1000
+
+#### volume deallocate
+volume.deallocate.enable=true
+volume.deallocate.workerNum=5
+volume.deallocate.batchClean=10
+volume.sdk.confPath=/etc/curvefs/curvebs_client.conf
```

Comment:
I think it's better to put service configure under service root directory like others:

```
volume.sdk.confPath=conf/curvebs_client.conf # __CURVEADM_TEMPLATE__ /curvefs/metaserver/conf/client.conf __CURVEADM_TEMPLATE__
```

Reply:
fix